app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -7,81 +7,58 @@
 'use strict';
 
 define([
-], function () {
+  'models/profile-image'
+], function (ProfileImage) {
 
   return {
     initialize: function (options) {
       this.notifications = options.notifications;
     },
 
-    // Attempt to load a profile image from the profile server
-    _fetchProfileImage: function (account) {
+    displayProfileImage: function (account, wrapperClass) {
```

Comment:
This is always called with an account, could the name be made more clear if called `displayAccountProfileImage`?